code/game/objects/items/weapons/manuals.dm

```diff
@@ -601,4 +601,309 @@
                 </ol>
 
 				</body>
-				</html>"}
\ No newline at end of file
+				</html>"}
+
+/obj/item/weapon/book/manual/virology_guide
+	name = "A Crash Course in Virology"
+	icon_state ="bookVirologyGuide"
+	author = "Frederick Chapman Montagnier"
+	title = "A Crash Course in Virology"
+	book_width = 819
+	book_height = 516
+	dat = {"<html>
```

Comment:
Can't all of this be put in a separate .css doc ? I'm asking because I don't see any way to do it myself.

Reply:
why this at all instead of a wiki ref? that can be formatted as we please too